src/streamlink_cli/argparser.py

```diff
@@ -1064,6 +1074,14 @@ def build_parser():
         Example: "aac"
         """
     )
+    transport.add_argument(
+        "--ffmpeg-no-start-at-zero",
+        action="store_true",
+        help="""
+        Disables the -start_at_zero ffmpeg option
+        when using copyts. 
```

Comment:
To make flake8 happy you need to remove the whitespace... 

```suggestion
        when using copyts.
```